types/classificator/index.d.ts

```diff
@@ -0,0 +1,120 @@
+// Type definitions for classificator 0.3
+// Project: https://github.com/Wozacosta/classificator
+// Definitions by: Nixinova <https://github.com/Nixinova>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace classificator;
+
```

Comment:
```suggestion
```
this is not an UMD package I think:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
it's just a plain, Node CommonJS module